src/pages/other-pension-providers/template.njk

```diff
@@ -0,0 +1,71 @@
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/details/macro.njk" import govukDetails %}
+
+{% extends "base.njk" %}
+
+{% block content %}
+    <h2 class="govuk-heading-l">Telling other pension providers</h2>
```

Comment:
I _think _ all the other headings are XL. Not totally sure which is correct, but probably best to be consistent